modules/ROOT/pages/index.adoc

```diff
@@ -18,14 +18,14 @@ security, and scalability.
 
 Anypoint Studio and Flow Designer support Mule application development.
 
-== Domains
+== Mule Domains
 
 Domains enable you to share global configurations that Mule applications need to
 reuse, such as default error handlers, shared properties, scheduler pools, and
-component configurations.
+component configurations. You can only deploy domains when running Mule runtime engine on-premises.
 
-Anypoint Studio supports Mule domain development.
-See xref:shared-resources.adoc[Shared Resources].
+See xref:shared-resources.adoc[Shared Resources] for Mule domain development and configuration using Anypoint Studio. +
+See xref:deploy-on-premises.adoc#deploy-a-domain[Deploy a Domain] for domain deployment instructions.
```

Comment:
Perhaps "See 'Deploy a Domain' for the steps involved in deploying domains."?
"Domain deployment instructions" is a noun string with three nouns in sequence, and noun strings always require more effort to parse than we should expect readers to expend. Plus, "domain deployment instructions" sounds bureaucratic and devoid of empathy.